test/parallel/test-process-binding.js

```diff
@@ -1,15 +1,16 @@
 'use strict';
+// Flags: --expose_internals
 var assert = require('assert');
 
 assert.throws(
   function() {
-    process.binding('test');
```

Comment:
Perhaps leave the `process.binding`s in this test so that the deprecated functionality is still tested before it is removed?